puppet/modules/openvox_repo/manifests/purge_puppet.pp

```diff
@@ -0,0 +1,5 @@
+class openvox_repo::purge_puppet() {
+  package {['puppet7-release', 'puppet8-release']:
```

Comment:
any chance you also had `puppet-release` and want to purge that as well?

Reply:
I am rather sure we have not, but it's cheap to purge that too.